packages/pluggableWidgets/background-gradient-native/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Breaking
+
+-   We added Atlas styling support for this widget. Regarding this change, you should update your `Background gradient` widgets' design properties. Please visit the [Mendix Docs](https://docs.mendix.com/refguide/native-styling-refguide/#1131-background-gradient) for more information on customization via Atlas.
```

Comment:
the link doesn't show the widget section for me, this is the one that worked `https://docs.mendix.com/refguide/mobile/designing-mobile-user-interfaces/widget-styling-guide/#1131-background-gradient`